stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/app.py

```diff
@@ -59,6 +60,10 @@
 filter_extension.conformance_classes.append(
     FilterConformanceClasses.ADVANCED_COMPARISON_OPERATORS
 )
+collection_search_extension = CollectionSearchExtension()
```

Comment:
Should we leave a comment explaining that the collection search extension is not implemented yet and the reason we are adding it here?